src/lib/utils/thorchain/lp/utils.ts

```diff
@@ -13,13 +13,13 @@ export function isLpConfirmedDepositQuote(
 }
 
 export function isLpConfirmedWithdrawalQuote(
-  quote: Record<string, unknown>,
+  quote?: Record<string, unknown>,
```

Comment:
this type is narrowed before being passed to any of the components so it should never be empty, however if we wanted to type it as if it could, we should use `| null` to match the top level type.

![image](https://github.com/user-attachments/assets/fdc76c11-c17b-4d86-be4d-51c905d1a0aa)

nitpick - use `Boolean()` for the `isLpConfirmedDepositQuote` as well

Reply:
Fixed in [b602e31](https://github.com/shapeshift/web/pull/7351/commits/b602e31990567afc87ae66b6a0c081c5b9bbdb26)